torch/csrc/jit/api/module.h

```diff
@@ -557,6 +557,7 @@ namespace script {
 // We once had a `script::` namespace that was deleted. This is for backcompat
 // of the public API; new code should not use this type alias.
 using Module = ::torch::jit::Module;
+using ExtraFilesMap = ::torch::jit::ExtraFilesMap;
```

Comment:
I don't see this used anywhere; was this intentional?

Reply:
Yes, it is for external users of the API, like in fbcode.